boxsdk/object/base_object.py

```diff
@@ -46,22 +30,14 @@ def __init__(self, session, object_id, response_object=None):
         :type response_object:
             :class:`BoxResponse`
         """
-        super(BaseObject, self).__init__(session)
```

Comment:
``super(BaseObject, self).__init__(session=session, response_object=response_object)`` should work, with the other `__init__` changes I mentioned. You should remove the manual `__init__` calls below.